File: tmp-mei/fgenjs5030319945734616766/base/ControlFlowContinue_1.js
import { base$$ControlFlow_1 } from "../base/ControlFlow_1.js";
import * as rt from "../rt/main.js";

export class base$$ControlFlowContinue_1 extends base$$ControlFlow_1 {
  static $self = new base$$ControlFlowContinue_1();

  match$mut(m_m$) {
  return base$$ControlFlowContinue_1.match$mut$fun(m_m$, this);
}

  static match$mut$fun(m_m$, $this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

}

File: tmp-mei/fgen4255027576711118260/base/ControlFlowContinue_1.java
package base;
public interface ControlFlowContinue_1 extends base.ControlFlow_1,base.Sealed_0{
ControlFlowContinue_1 $self = new ControlFlowContinue_1Impl();
Object match$mut(base.ControlFlowMatch_2 m_m$);
static Object match$mut$fun(base.ControlFlowMatch_2 m_m$, base.ControlFlowContinue_1 $this) {
  return (switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (Object) null;
}})
;
}
}